src/docker/composebuilder/localdependencies/helpers.go

```diff
@@ -0,0 +1,26 @@
+package localdependencies
```

Comment:
It doesn't need to be, but it seemed like we're conflating too many things in one package. Just because `composebuilder` is the only package that uses this local dependency stuff doesn't mean they should be lumped into the same package. I would even be open to extracting a `dependency` package and putting `localdependencies` and `remotedependencies` in there.

Reply:
I think this `LocalDependency` strut should actually be removed in favor a `DependencyBuilder` struct. The different between local and remote dependencies doesn't matter to `composebuilder` since its only concerned with local

Within the `types` package, we could move LocalDependency and RemoteDependency into a dependencies subpage